variables.tf

```diff
@@ -296,6 +296,18 @@ variable "operator_image_id" {
   type        = string
 }
 
+variable "operator_volume_kms_id" {
+  default     = ""
+  description = "The OCID of the OCI KMS key to assign as the master encryption key for the boot volume."
+  type        = string
+}
+
+variable "enable_operator_pv_encryption_in_transit" {
+  default     = false
+  description = "Whether to enable in-transit encryption for the data volume's paravirtualized attachment."
+  type        = bool
```

Comment:
Please add validation

Reply:
what do you mean by validation here ? enable_operator_pv_encryption_in_transit is independent on other features and it can be either true or false which is already captured by bool type

Reply:
see here for an example: https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/main/variables.tf#L223